recipes/mpi-serial/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "mpi-serial" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MCSclimate/mpi-serial/archive/refs/tags/MPIserial_{{ version }}.tar.gz
+  sha256: cc55e6bf0ae5e1d93aafa31ba91bfc13e896642a511c3101695ea05eccf97988
+  patches:
+    - 0001-Rearrange-code-and-other-minor-fixes-needed-for-comp.patch
+    - fix-makefile.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    # Unix
+    - {{ compiler('c') }}  # [unix]
+    - {{ compiler('cxx') }}  # [unix]
+    - {{ compiler('fortran') }}  # [unix]
+    - coreutils  # [unix]
+    - make  # [unix]
+    # Win
+    - {{ compiler('m2w64_c') }}  # [win]
+    - {{ compiler('m2w64_cxx') }}  # [win]
+    - {{ compiler('m2w64_fortran') }}  # [win]
+    - m2-autoconf  # [win]
+    - m2-automake-wrapper  # [win]
+    - m2-coreutils  # [win]
+    - m2-grep  # [win]
+    - m2-libtool  # [win]
+    - m2-m4  # [win]
+    - m2-make  # [win]
+    - m2-sed  # [win]
+    # - m2-which  # [win]
+    # - m2w64-pkg-config  # [win]
+    # - m2w64-dlfcn  # [win]
+  host:
+  run:
```

eventually:
```suggestion
  run:
      mpi 1.0 mpi-serial
```
or similar.

---

Based on MS-MPI, we would merge this without the `mpi` dependency:
https://github.com/conda-forge/staged-recipes/pull/12751/files#diff-6729109fd86db8185b3faf24fb32ca9f3f409dcabd98ed85bc99720ffab83d9d
and then add it later:
https://github.com/conda-forge/msmpi-feedstock/pull/1/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9a

---

No, there's no point adding it later

---

Thanks @isuruf, I guess that means
https://github.com/conda-forge/mpi-feedstock/pull/10#issuecomment-1546772523
would need to get merged first, then.

---

@zklaus, this can be added now that https://github.com/conda-forge/mpi-feedstock/pull/10 has gone in.